gallery/visualtests/nxSegmentedButton.js

```diff
@@ -61,10 +64,17 @@ describe('NxSegmentedButton', function() {
       it('has a darker blue background when hovered', hoverTest(selector, dropdownBtnSelector));
       it('has a lighter blue background when clicked', clickTest(selector, dropdownBtnSelector));
       it('has a white inner outline when focused', focusTest(selector, dropdownBtnSelector));
-      it('has a lighter blue background and white inner outline when opened',
-          openedTest(selector, dropdownBtnSelector));
-      it('has a lighter blue background when opened and not focused',
-          openedTest(selector, dropdownBtnSelector, dropdownMenuItemSelector));
+
+      describe('when open', function() {
+        it('has a lighter background and white inner outline when focused',
+            openedTest(selector, dropdownBtnSelector));
+        it('has a lighter background when not focused',
+            openedTest(selector, dropdownBtnSelector, dropdownMenuItemSelector));
+        it('has a darker background when hovered',
+            openedTest(selector, dropdownBtnSelector, null, true));
+        it('has a darker background and white inner outline when hovered and focused',
```

Comment:
These tests seem to be intended to check when the toggle button is hovered and focused, but the screenshots show the toggle button hovered while a dropdown menu item is focused

Reply:
> These tests seem to be intended to check when the toggle button is hovered and focused, but the screenshots show the toggle button hovered while a dropdown menu item is focused

I realize I mixed up the two tests- hover + focus when only supposed to be hovered, and vice versa. When I wrote these tests during the interaction states epic I shifted focus to the dropdown menu to remove focus from the dropdown button- should I just blur the focus instead and remove any focus outlines?

Reply:
>  These tests seem to be intended to check when the toggle button is hovered and focused, but the screenshots show the toggle button hovered while a dropdown menu item is focused

As discussed in WW, the focus outlines were removed and snapshots were updated to reflect the proper interaction states. 
Segmented Button: [e5ad2f6](https://github.com/sonatype/sonatype-react-shared-components/pull/1120/commits/e5ad2f68e91ece2accdb3d4983ce696c6ee19e33)
Icon Dropdown: [5dac819](https://github.com/sonatype/sonatype-react-shared-components/pull/1120/commits/5dac819a83a215f5a709836db0671e501352dfdf)